### DISCUSSION TITLE
QoL task update

### DISCUSSION THREAD

**Discussion by RainVaporeon** [DiscordDungeons/Bugs#979] [Ideas]
When accepting a task, you should also be able to talk to Alex again to view your task (and an option to cancel it, not changing to a new one because you know, their fault agreeing to it)
When in a task, talking to the NPC without sufficient materials/items would result in them having a different dialogue, such as:
"Have you gotten <item> yet?" "I need <item>, can you help me get them?" .,etc. so people would know that they do not have enough of the items to complete the task.

**Comment by FelixArgyle** [comment]
Uhm you do know that there is a `#!task check` option right? 
![example](http://senpai.pls-fuck.me/6ZeXkQa.jpg)

**Reply by Ciborn** [reply]
For more role-playing, this could actually be a good idea, but not as a replacement for `#!task check`. Aside from role-playing, it would be useless as we already have `#!task check`.

**Comment by Sven65** [comment]
There's a way of doing this already.
